### PR TITLE
fix: ticket sales value 

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/5631

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/helpers/round-up.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import Helper from '@ember/component/helper';
+import { ceil } from 'lodash-es';
+
+export function roundUp(params) {
+  return ceil(params[0], 2);
```
Comment (iamareebjamal) [inline app/helpers/round-up.js:5]: Use Math.ceil

**Comment by iamareebjamal** [comment]
We already have ember math helpers, use round helper from that

**Comment by maze-runnar** [comment]
using `ceil` helper.

**Comment by iamareebjamal** [comment]
It's not displaying decimals now

**Comment by maze-runnar** [comment]
> It's not displaying decimals now

round is not also displaying decimals. For that I have to use custom helper.

**Comment by iamareebjamal** [comment]
No, you can pass `decimal=2` in the round helper. Check the docs

**Comment by iamareebjamal** [comment]
I'm seeing only 1 digit after decimal

**Comment by maze-runnar** [comment]
> I'm seeing only 1 digit after decimal

if there are more than two or two digits after decimal then it will show two digits after decimal. Otherwise wise original number

**Pull request merged**